tests/testthat/example_default_load/dettl.yml

```diff
@@ -0,0 +1,24 @@
+sources:
+  - R/extract.R
+  - R/transform.R
+  - R/verification_queries.R
+
+extract:
+  func: extract
+  test: R/test_extract.R
+
+transform:
+  func: transform
+  test: R/test_transform.R
+
+load:
+  verification_queries: test_queries
+  test: R/test_load.R
+
+default_load: TRUE
```

Comment:
should this go into the `load` block as a key `default: true`?

Reply:
and I think the config validation needs a  check that if the default load is given then `load: func` is missing or empty